index.bs

```diff
@@ -7992,9 +7989,9 @@ a reference to the same object that the IDL value represents.
         starting at byte offset |offset|.
 </div>
 
-<div algorithm="detach an arraybuffer">
+<div algorithm>
```

Comment:
Yes, the algo name is currently only used to show error messages.  When I start adding in reified algorithms, they'll be how you declare that you're calling a particular algorithm.